action.yml

```diff
@@ -2,9 +2,42 @@
 name: 'Rubocop Linter Action'
 description: 'A GitHub Action that lints your Ruby code with Rubocop!'
 author: 'Andrew Mason <andrewmcodes@protonmail.com>'
+inputs:
+  version:
+    description: 'Define a specific version of Rubocop otherwise uses latest'
+    required: false
+    default: ''
+  additional_gems:
+    description: 'Define additional gems to install to use with rubocop'
+    required: false
+    default: 'rubocop-performance rubocop-rails rubocop-minitest rubocop-rspec'
+  config_path:
+    description: 'Define a path to a config file not in the ./ named .rubocop.yml'
+    required: false
+    default: ''
+  excluded_cops:
+    description: 'List out Cops or departments to exclude from run'
+    required: false
+    default: ''
+  file_paths:
+    description: 'List File paths to inspect, default is all files not excluded in .rubocop.yml file'
+    required: false
+    default: ''
+  fail_level:
+    description: 'Define Fail level, available values are refactor, convention, warning, error and fatal'
+    required: false
+    default: 'warning'
+  bundle:
+    description: 'Useful gemfile, useful for using your oen version of the gem'
```

Comment:
Typo.

Also maybe instead of ‘Use’ let’s say ‘Bundle’

Reply:
Fixed the typo with a better verbiage